### PR TITLE
Themes webview test

### PR DESCRIPTION
**Pull request by holiber**

### CODE REVIEW

**Comment by blackxored** [comment]
Can we rebase this into 1 or 2 commits?

**Comment by holiber** [comment]
We always do `Squash and Merge` so all commits will be squashed into one anyway

**Comment by holiber** [comment]
You can merge commits from here into your original PR if you wish

File: test/helpers/spectron/index.ts [hunk 1]
```diff
@@ -32,6 +33,13 @@ export async function focusChild(t: any) {
   await focusWindow(t, /windowId=child/);
 }
 
+// Focuses the Library webview
+export async function focusLibrary(t: any) {
+  // doesn't work without delay, probably need to wait until load
+  await sleep(2000);
+  await focusWindow(t, /streamlabs\.com\/library/);
```
Comment (andycreeth) [inline test/helpers/spectron/index.ts]: Wow.  I'm surprised this works.  Nice.

**Pull request merged**